tutorials/3d/fps_tutorial/part_one.rst

```diff
@@ -312,7 +312,7 @@ so it can move through the game world.
 
 _________
 
-Let's look is ``process_movement`` next:
+Let's look is ``process_input`` next:
```

Comment:
Two more typos to fix on this line IMO:
- "look is" -> "look at"
- "next" -> "first"

Reply:
Right for "look at", but this is the third point is the description, so let's keep "next" ;)